templates/alloy-template/src/components/base/Header.tsx

```diff
@@ -0,0 +1,124 @@
+import Link from 'next/link';
+
+interface NavigationItem {
+  label: string;
+  href: string;
+}
+
+interface HeaderProps {
+  currentPath: any;
+  navigationItems?: NavigationItem[];
+  logoText?: string;
+}
+
+const defaultNavigationItems: NavigationItem[] = [
```

Comment:
Now when we have `getItems()` we should be able to load these items from the CMS.